### PR TITLE
Use more descriptive link texts

### PR DESCRIPTION
**Pull request by remigathoni**
Hey @TomNicholas! This closes issue #7617, please review. Thanks!

### CODE REVIEW

File: doc/contributing.rst [hunk 1]
```diff
@@ -732,8 +732,8 @@ or, to use a specific Python interpreter,::
 This will display stderr from the benchmarks, and use your local
 ``python`` that comes from your ``$PATH``.
 
-Information on how to write a benchmark and how to use asv can be found in the
-`asv documentation <https://asv.readthedocs.io/en/latest/writing_benchmarks.html>`_.
+Learn `how to write a benchmark and how to use asv from the documentayion <https://asv.readthedocs.io/en/latest/writing_benchmarks.html>`_ .
```
Comment (headtr1ck) [inline doc/contributing.rst:735]: documentation

File: doc/user-guide/dask.rst [hunk 2]
```diff
@@ -549,7 +549,7 @@ larger chunksizes.
 
 .. tip::
 
-   Check out the dask documentation on `chunks <https://docs.dask.org/en/latest/array-chunks.html>`_.
+   Check out the `dask documentation of chunks <https://docs.dask.org/en/latest/array-chunks.html>`_.
```
Comment (headtr1ck) [inline doc/user-guide/dask.rst:552]: Not a native speaker but "on" sounds better to me?

**Review by headtr1ck — APPROVED** [review]
Nice changes.
Allows more natural reading.

**Comment by headtr1ck** [comment]
Could you also merge main again?

**Comment by dcherian** [comment]
Thanks @remigathoni This is a great first contribution. Welcome to Xarray! We're excited to see your next contribution :)

**Comment by remigathoni** [comment]
> Thanks @remigathoni This is a great first contribution. Welcome to Xarray! We're excited to see your next contribution :)

Thank you @dcherian !

**Pull request merged**